CHANGELOG.md

```diff
@@ -7,6 +7,10 @@
     Click to see more.
```

Comment:
I'd maybe hide the `Unreleased` section with `<!-- -->` until there is something there to show.